addons/mail/static/src/models/file_uploader.js

```diff
@@ -16,7 +16,7 @@ const getAttachmentNextTemporaryId = (function () {
 
 registerModel({
     name: 'FileUploader',
-    identifyingFields: [['activityView', 'attachmentBoxView', 'composerView']],
+    identifyingFields: [['activityView', 'attachmentBoxView', 'chatterOwner', 'composerView']],
```

Comment:
`attachmentBoxView` field can be removed everywhere from file uploader if if is no longer used

Reply:
We are still using it when the user clicks on `Attach files` inside `attachmentBoxView`

![Screen Shot 2022-06-01 at 3 39 11 PM](https://user-images.githubusercontent.com/21078434/171381124-1b17fcc8-9253-4423-9793-adae8868d014.png)
`

Reply:
Oh right, couldn't we use the one from chatter in that case too to clean a little bit?
But I guess it could be done as a follow up PR to clean, and not necessarily here.